test/integration/promise.test.js

```diff
@@ -10,7 +10,7 @@ var chai = require('chai')
 
 chai.config.includeStack = true;
 
-describe(Support.getTestDialectTeaser('Promise'), function() {
+describe.skip(Support.getTestDialectTeaser('Promise'), function() {
```

Comment:
Does it log a warning when using it? If so we can remove it for 2.1, if not we need to add the warning and then we can remove in 2.2.